doc/htmldoc/installation/cmake_options.rst

```diff
@@ -87,6 +87,11 @@ External libraries
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
 | ``-Dwith-gsl=[OFF|ON|</path/to/gsl>]``                | Build with the GSL library [default=ON]. To set a specific library, give the install path.     |
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
++-------------------------------------------------------+------------------------------------------------------------------------------------------------+
+| ``-Dwith-hdf5=[OFF|ON|</path/to/hdf5>]``              | Build with `HDF5 <https://hdfgroup.org/>`_ library [default=OFF]. To set a specific library,   |
+|                                                       | give the install path. HDF5 is required for                                                    |
+|                                                       | `SONATA <https://github.com/AllenInstitute/sonata>`_ support.                                  |
```

Comment:
Just one last little thing: I think this link should go to our `nest_sonata_guide.rst` to provide more context for the user. The general Sonata home page will not help to make the connection to this compiler option.